package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@epages/react-components",
-  "version": "0.6.1",
+  "version": "0.6.2",
```

Comment:
Please update or rebase this branch onto "master".
It looks like this PR is based on an old state of the "master" branch (the current version in "master" is `0.9.0`).